np10.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def sort_axis(array):
+    """
+    Function that sort an along the first, last axis of a given array
+    """
+    first_axis = np.sort(array, axis=0)
+    print(f"sorted by 1st axis: \n {first_axis}")
```

Comment:
Seperate IO from logic -
 don’t use print in functions unless there’s a good reason